org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
```

Comment:
Yes, this version has been developed and mostly tested on 3.22, only in current master we support 3.24 (fixed minor warning and stuff).